scapy/arch/windows/__init__.py

```diff
@@ -539,7 +540,7 @@ def sndrcv(pks, pkt, timeout = 2, inter = 0, verbose=None, chainCC=0, retry=0, m
                                     if r.answers(sentpkt):
                                         ans.append((sentpkt, r))
                                         if verbose > 1:
-                                            os.write(1, "*")
+                                            sys.stdout.write("*")
```

Comment:
Understood, but that would be another PR then.

Maybe an option for that would be to create a `conf.output` value and use it everywhere? That would be good to have @guedou's opinion here.